### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by zenseii**
This adds new strings and adjusts some according to recent code changes.

### CODE REVIEW

File: files/lang/fr.po [hunk 1]
```diff
@@ -570,29 +568,24 @@ msgstr "%{spell} inflige %{value} dégâts à une créature vivante."
 msgid "The %{spell} does %{damage} damage to all living creatures."
 msgstr "%{spell} inflige %{value} dégâts à toutes les créatures vivantes."
 
-#, fuzzy
 msgid "The %{attacker}' attack blinds the %{target}!"
-msgstr "L'attaque des Licornes aveugle les %{name} !"
+msgstr "L'attaque de %{attacker} aveugle les %{target} !"
```
Comment (zenseii) [inline files/lang/fr.po:572]:
```suggestion
msgstr "L'attaque de la troupe de %{attacker} aveugle la troupe de %{target} !"
```

File: files/lang/fr.po [hunk 2]
```diff
@@ -2607,9 +2596,8 @@ msgstr "Change le volume des effets sonores."
 msgid "Change the type of music."
 msgstr "Change de type de musique."
 
-#, fuzzy
 msgid "Toggle 3D effects of foreground sounds."
-msgstr "Change le volume des effets sonores."
+msgstr "Basculer les effets 3D des sons d'avant-plan."
```
Comment (zenseii) [inline files/lang/fr.po:2600]:
```suggestion
msgstr "Basculer les effets 3D des sons au premier plan"
```

File: files/lang/fr.po [hunk 3]
```diff
@@ -2663,33 +2651,28 @@ msgid "Settings"
...
+msgstr "Noir et blanc"
 
-#, fuzzy
 msgid "Mouse Cursor"
-msgstr "Ombre du curseur"
+msgstr "Curseur de la sourcis"
 
 msgid "Color"
-msgstr ""
+msgstr "Couleuré"
```
Comment (zenseii) [inline files/lang/fr.po:2672]:
```suggestion
msgstr "Coloré"
```

File: files/lang/fr.po [hunk 4]
```diff
@@ -2735,7 +2717,7 @@ msgid "Toggle instant battle mode."
 msgstr "Change les modes de combat instantané."
 
 msgid "Check all Hot Keys used in the game."
-msgstr ""
+msgstr "Verifier tous les raccourcirs clavier du jeu."
```
Comment (zenseii) [inline files/lang/fr.po:2720]:
```suggestion
msgstr "Verifier les raccourcirs clavier du jeu."
```

File: files/lang/fr.po [hunk 5]
```diff
@@ -2744,16 +2726,20 @@ msgid ""
 "Toggle color cursors on/off. Color cursors look nicer, but sometimes don't "
 "move as smoothly as black and white ones."
 msgstr ""
+"Basculer le curseur de la souris couleuré. Les curseurs couleurés sont plus "
```
Comment (zenseii) [inline files/lang/fr.po:2729]:
```suggestion
"Basculer le curseur en couleur. Les curseurs colorés sont plus "
```

File: files/lang/fr.po [hunk 6]
```diff
@@ -2744,16 +2726,20 @@ msgid ""
...
+"beaux, mais parfois ils ne se déplacent pas fluide comme ceux en noir et "
```
Comment (zenseii) [inline files/lang/fr.po:2730]:
```suggestion
"beaux, mais peuvent ne pas être aussi fluide que celui en noir et "
```

File: files/lang/fr.po [hunk 7]
```diff
@@ -2744,16 +2726,20 @@ msgid ""
...
+"blanc."
 
 msgid ""
 "Toggle text support mode to output extra information about windows and "
 "events in the game."
 msgstr ""
+"Basculer la mode texte pour montrer des informations en plus sur les "
```
Comment (zenseii) [inline files/lang/fr.po:2737]:
```suggestion
"Activer le mode texte pour afficher des informations supplémentaires sur les différentes "
```

File: files/lang/fr.po [hunk 8]
```diff
@@ -3539,13 +3521,12 @@ msgstr "L'équipe fheroes2 Resurrection présente"
...
 msgstr "Salutations !"
 
-#, fuzzy
 msgid ""
 "Welcome to Heroes of Might and Magic II powered by fheroes2 engine! Before "
 "starting the game please choose game resolution."
 msgstr ""
-"Bienvenue à fheroes2 ! Avant de commencer le jeu choisissez la résolution du "
-"jeu."
+"Bienvenue à Heroes of Might and Magic II ! Avant de commencer le jeu "
```
Comment (zenseii) [inline files/lang/fr.po:3528]:
```suggestion
"Bienvenue sur Heroes of Might and Magic II (amélioré par le moteur fheroes2) ! Avant de commencer le jeu "
```

File: files/lang/fr.po [hunk 9]
```diff
@@ -570,29 +568,24 @@ msgstr "%{spell} inflige %{value} dégâts à une créature vivante."
...
-#, fuzzy
 msgid "The %{target} are paralyzed by the %{attacker}!"
-msgstr "%{name} sont paralysés par les Cyclopes !"
+msgstr "%{target} sont paralysés par %{attacker} !"
 
-#, fuzzy
 msgid "The %{attacker} dispel all good spells on your %{target}!"
-msgstr "Les Grands mages dissipent tous les bons sorts sur vos %{name} !"
+msgstr ""
+"Le groupe de %{attacker} dissipe tous les bons sorts sur vos %{target} !"
```
Comment (zenseii) [inline files/lang/fr.po:585]:
```suggestion
"La troupe de %{attacker} dissipe tous sorts positives sur vos %{target} !"
```

File: files/lang/fr.po [hunk 10]
```diff
@@ -570,29 +568,24 @@ msgstr "%{spell} inflige %{value} dégâts à une créature vivante."
...
 
-#, fuzzy
 msgid "The %{attacker}' curse falls upon the %{target}!"
-msgstr "La malédiction des Momies s'abat sur les %{name} !"
+msgstr "La malédiction de groupe de %{attacker} s'abat sur les %{target} !"
 
-#, fuzzy
 msgid "The %{target} are paralyzed by the %{attacker}!"
-msgstr "%{name} sont paralysés par les Cyclopes !"
+msgstr "%{target} sont paralysés par %{attacker} !"
```
Comment (zenseii) [inline files/lang/fr.po:581]:
```suggestion
msgstr "La troupe de %{target} est paralysée par %{attacker} !"
```

File: files/lang/fr.po [hunk 11]
```diff
@@ -570,29 +568,24 @@ msgstr "%{spell} inflige %{value} dégâts à une créature vivante."
...
 
-#, fuzzy
 msgid "The %{attacker}' attack blinds the %{target}!"
-msgstr "L'attaque des Licornes aveugle les %{name} !"
+msgstr "L'attaque de la troupe de %{attacker} aveugle la troupe de %{target} !"
 
-#, fuzzy
 msgid "The %{attacker}' gaze turns the %{target} to stone!"
-msgstr "Le regard des Méduses transforme les %{name} en pierre !"
+msgstr "Le regard de %{attacker} transforme les %{target} en pierre !"
```
Comment (zenseii) [inline files/lang/fr.po:575]:
```suggestion
msgstr "Le regard de la troupe de %{attacker} transforme la troupe de %{target} en pierre !"
```

File: files/lang/fr.po [hunk 12]
```diff
@@ -17,7 +17,7 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n > 1;\n"
 "X-Launchpad-Export-Date: 2010-05-27 09:12+0000\n"
-"X-Generator: Poedit 3.0.1\n"
+"X-Generator: Poedit 3.1\n"
```
Comment (ihhub) [inline files/lang/fr.po:20]: Why did we change this? I assume that our pipeline will change it back.

Comment (zenseii) [reply files/lang/fr.po:20]: It's automatically changed because I updated poedit on my setup.

Comment (zenseii) [reply files/lang/fr.po:20]: Also this is irrelevant to this comment, but I'm awaiting feedback for a few strings before I mark this as ready for final review.

**Comment by ihhub** [comment]
Hi @zenseii , what is the state of this pull request?

File: files/lang/fr.po [hunk 13]
```diff
@@ -1592,12 +1584,11 @@ msgstr ""
...
 "du scénario."
 
-#, fuzzy
 msgid ""
 "The kingdom will have %{amount} less %{resource} at the start of the "
 "scenario."
 msgstr ""
-"Le royaume obtiendra des ressources supplémentaires en %{resource} au début "
+"Le royaume obtiendra %{amount} ressources de moins en %{resource} au début "
 "du scénario."
```
Comment (zenseii) [inline files/lang/fr.po:1592]:
```suggestion
"Le royaume aura %{amount} ressource en %{resource} de moins au début "
"du scénario."
```

**Comment by zenseii** [comment]
Hi @ihhub! This PR is ready for your review when you feel you have the time and energy to.

**Comment by ihhub** [comment]
@zenseii , huge thanks for maintaining translations!

**Pull request merged**